src/test/java/io/jenkins/pluginhealth/scoring/scores/PluginMaintenanceScoringTest.java

```diff
@@ -46,135 +47,234 @@
-            DependabotPullRequestProbe.KEY, new ProbeResult(DependabotPullRequestProbe.KEY, "0", ResultStatus.SUCCESS)
-        ));
-
-        final ScoreResult result = scoring.apply(plugin);
-        assertThat(result.key()).withFailMessage(() -> "Score key should be '%s'".formatted(KEY)).isEqualTo(KEY);
-        assertThat(result.coefficient()).withFailMessage(() -> "Score coefficient should be '%f'".formatted(COEFFICIENT)).isEqualTo(COEFFICIENT);
-        assertThat(result.value()).isEqualTo(.9f);
-    }
-
-    @Test
-    public void shouldScoreNinetyForPluginsWithJenkinsfileAndDependabotButNoJEP229() {
-        final Plugin plugin = mock(Plugin.class);
-        final PluginMaintenanceScoring scoring = spy(PluginMaintenanceScoring.class);
-
-        when(plugin.getDetails()).thenReturn(Map.of(
-            JenkinsfileProbe.KEY, new ProbeResult(JenkinsfileProbe.KEY, "", ResultStatus.SUCCESS),
-            DependabotProbe.KEY, new ProbeResult(DependabotProbe.KEY, "", ResultStatus.SUCCESS),
-            DependabotPullRequestProbe.KEY, new ProbeResult(DependabotPullRequestProbe.KEY, "0", ResultStatus.SUCCESS),
-            ContinuousDeliveryProbe.KEY, new ProbeResult(ContinuousDeliveryProbe.KEY, "", ResultStatus.FAILURE)
-        ));
-
-        final ScoreResult result = scoring.apply(plugin);
-        assertThat(result.key()).withFailMessage(() -> "Score key should be '%s'".formatted(KEY)).isEqualTo(KEY);
-        assertThat(result.coefficient()).withFailMessage(() -> "Score coefficient should be '%f'".formatted(COEFFICIENT)).isEqualTo(COEFFICIENT);
-        assertThat(result.value()).isEqualTo(.9f);
-    }
-
-    @Test
-    public void shouldScoreHundredForPluginsWithJenkinsfileAndDependabotAndJEP229() {
-        final Plugin plugin = mock(Plugin.class);
-        final PluginMaintenanceScoring scoring = spy(PluginMaintenanceScoring.class);
-
-        when(plugin.getDetails()).thenReturn(Map.of(
-            JenkinsfileProbe.KEY, new ProbeResult(JenkinsfileProbe.KEY, "", ResultStatus.SUCCESS),
-            DependabotProbe.KEY, new ProbeResult(DependabotProbe.KEY, "", ResultStatus.SUCCESS),
-            DependabotPullRequestProbe.KEY, new ProbeResult(DependabotPullRequestProbe.KEY, "0", ResultStatus.SUCCESS),
-            ContinuousDeliveryProbe.KEY, new ProbeResult(ContinuousDeliveryProbe.KEY, "", ResultStatus.SUCCESS)
-        ));
-
-        final ScoreResult result = scoring.apply(plugin);
-        assertThat(result.key()).withFailMessage(() -> "Score key should be '%s'".formatted(KEY)).isEqualTo(KEY);
-        assertThat(result.coefficient()).withFailMessage(() -> "Score coefficient should be '%f'".formatted(COEFFICIENT)).isEqualTo(COEFFICIENT);
-        assertThat(result.value()).isEqualTo(1f);
-    }
+        public static final String KEY = "repository-configuration";
+        public static final float COEFFICIENT = .5f;
+
+        @Test
+        public void shouldScoreZeroWhenNoJenkinsfileProbeResult() {
```

Comment:
It's not only the Jenkinsfile that is missing in this case.
You could name that test as `shouldScoreZeroWithNoRepositoryToolsConfigured` or something like that.